libs/datamodel/core/src/ast/parser/datamodel.pest

```diff
@@ -63,7 +63,7 @@ enum_value_declaration = { doc_comment_and_new_line* ~ (non_empty_identifier ~ (
 // ######################################
 // Directives
 // ######################################
-block_level_directive = { "@@" ~ directive ~ NEWLINE }
+block_level_directive = { "@@" ~ directive ~ doc_comment? ~ NEWLINE }
```

Comment:
Will that work with a normal `//` comment as well?

Reply:
Yes, normal comments were not a problem before either.

Reply:
I've added a test for this now, the `comment?` was missing here.